### DISCUSSION TITLE
Filebeat module

### DISCUSSION THREAD

**Discussion by bcmcevoy** [Security-Onion-Solutions/securityonion#1778] [Unsupported Versions] [answered]
There doesn't seem to be a option for Filebeat module's in the security onion installation. If I want to enable a filebeat module do I have to download install it from elastic, and if so how do I import it into the salt directory.

**Comment by weslambert** [comment, marked as answer]
If it must occur outside of modification of the config file, enabling additional non-default modules is not currently supported for Filebeat.

**Comment by bcmcevoy** [comment]
No the module folder itself comes default with the Filebeat download from their website. Furthermore this one only modifies the config folder to fix the parsing for certain logs, i.e. sonicwall. For some reason security onion's version of Filebeat did not come with the module folder, I'm not exactly sure why. Is there any way to add the default filebeat module folder into Security Onion? 
        Would that have to be something on your end or if I can do it, how would I do it exactly, I can install the folder no problem but at that point its not in the salt stack and I am not sure how to add it to the so filebeat install directory.

**Reply by weslambert** [reply]
You can copy `/opt/so/saltstack/default/salt/filebeat/init.sls` to `/opt/so/saltstack/local/salt/filebeat/init.sls` and modify the Docker binds as needed.

**Reply by Grey999** [reply]
Is Suricata a default module of filebeat ? I have some trouble parsing eve.log and I thought it could be link to that.

**Reply by dougburks** [reply]
@Grey999 From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
